app/livechat/server/api/v1/offlineMessage.js

```diff
@@ -10,10 +10,11 @@ API.v1.addRoute('livechat/offline.message', {
 				name: String,
 				email: String,
 				message: String,
+				department: String,
```

Comment:
This should be optional,  otherwise we can't send offline messages without `department`.